orte/mca/rml/ofi/rml_ofi_component.c

```diff
@@ -332,12 +403,13 @@ static int orte_rml_ofi_query_transports(opal_value_t **providers)
     }
 
     opal_output_verbose(10,orte_rml_base_framework.framework_output,
-                        "\n%s:%d Completed Query Interface",__FILE__,__LINE__);
+                        "\n%s:%d Completed Query Interface",__FILE__,__LINE__);*/
     return ORTE_SUCCESS;
 }
 
 
 /*debug routine to print the opal_value_t returned by query interface */
+/* [Anandhi] - this has to be re-written
```

Comment:
Please delete old code that is no longer being used.  I'll stop saying that now 😄 , but it applies to any other commented-out code in this PR.